src/main/ngapp/accio/src/app/editor/kdTree.js

```diff
@@ -0,0 +1,465 @@
+/**
```

Comment:
You don't need to break up this PR.  It is better to add third party dependencies in a standalone PR. This separates packages from code.

Additionally, the best thing to do is to commit the unmodified library.  Then submit an additional PR if you need to make any modifications to the library to make it compile/work in your repository.  Your future self will thank you when you need to the exact same task again.

Reply:
Sounds good, Thanks!